tt_llk_wormhole_b0/common/inc/ckernel_mutex_guard.h

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
```

Comment:
```suggestion
#include "ckernel.h"
```
This is needed for `t6_mutex_acquire/t6_mutex_release`